cip25/src/lib.rs

```diff
@@ -1,91 +1,86 @@
-use std::io::{BufRead, Write};
```

Comment:
The diff on all these files are really hard to read since it looks like cddl-codegen doesn't have a stable order for its output. Probably we should change that so diffs are clean in the future